class/hd-segwit-bech32-wallet.js

```diff
@@ -22,12 +23,23 @@ export class HDSegwitBech32Wallet extends AbstractHDWallet {
   static typeReadable = 'HD SegWit';
   static defaultRBFSequence = 2147483648; // 1 << 31, minimum for replaceable transactions as per BIP68
   static randomBytesSize = 16;
-  static basePath = "m/84'/440'/0'";
 
-  constructor() {
+  constructor({ isElectrumVault } = {}) {
```

Comment:
since the object here has only one member so far wouldn't It better to have it simply as a optional plain parameter, not as an object?

Reply:
doesn't really matter